docs/docfx.json

```diff
@@ -44,8 +44,8 @@
       "ms.author": "o365devx",
       "author": "o365devx",
       "ms.topic": "conceptual",
-      "ms.prod": "office",
-      "ms.technology": "office-add-ins"
+      "ms.prod": "Office 365",
+      "ms.technology": "Add-ins"
```

Comment:
Doesn't this go against the values the CATS tool was scanning for?

Reply:
That's a good question. We're trying to get CATS, APIScan, PowerBI, and our docsets to all use the same taxonomy that we use. I'll ask Kelly but I do think she has "Office 365" whitelisted since she asked me to do this. Stay tuned! FYI, here is our taxonomy list:
https://microsoft.sharepoint.com/teams/OfficePlatform/Lists/Taxonomy/Taxonomy%20sorted%20by%20vocabulary.aspx

Reply:
Did we get an answer on this?

Reply:
Yes, per Kelly, we want our taxonomy to sync with CATS and SkyEye. So dropping "office-" from ms.technology and adding " 365" to ms.prod. (See Taxonomy list in Teams SharePoint site, https://microsoft.sharepoint.com/teams/OfficePlatform/Lists/Taxonomy/Taxonomy%20sorted%20by%20vocabulary.aspx)